index.html

```diff
@@ -0,0 +1,468 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="stylesheet" href="./css/header.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <link
+      href="https://unpkg.com/boxicons@2.0.7/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+
+      <div class="md:hidden mob_box">
+        <div id="myNav" class="overlay-header">
+          <button class="closebtn" onclick="closeNav()">
+            <i class="bx bx-x"></i>
+          </button>
+          <div class="overlay-content">
+            <a href="#home" class="header__link">Portfolio</a>
+            <hr class="nav-hr" />
+            <a href="#about" class="header__link">About</a>
+            <hr class="nav-hr" />
+            <a href="#connect" class="header__link">Contact</a>
+            <hr class="nav-hr" />
+          </div>
+        </div>
+        <button type="button" class="menu-bars" onclick="openNav()">
+          <i class="fas fa-bars"></i>
+        </button>
+      </div>
```

Comment:
Kindly wrap the mobile menu links inside the `nav` tag instead of `div`. The nav element is semantically more specific, so the search engine will understand that the links within it represent navigation. For more information, you can have a look at this [article](https://www.w3.org/community/webed/wiki/Creating_multiple_pages_with_navigation_menus)